daemon/oci_windows.go

```diff
@@ -25,11 +25,30 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	// In base spec
 	s.Hostname = c.FullHostname()
 
+	if err := daemon.setupConfigDir(c); err != nil {
+		return nil, err
+	}
+
 	// In s.Mounts
 	mounts, err := daemon.setupMounts(c)
 	if err != nil {
 		return nil, err
 	}
+
+	if len(c.ConfigReferences) > 0 {
+		if err := daemon.Mount(c); err != nil {
```

Comment:
This is another case of a relatively expensive mount/unmounts which should be avoided if possible.